### ISSUE TITLE
Notification message on error doesn't make sense

### ISSUE THREAD

**Issue by arai-a** [mozilla-services/screenshots#2896]
(the reproduction steps are similar to #2838)

Steps to reproduce:
  1. open firefox nightly with clean profile
  2. open https://screenshots.firefox.com/
  3. click Firefox Screenshots button on toolbar
  4. click "SKIP" to skip tutorial
  5. hit "ESC" to escape the Screenshots
  6. click Firefox Screenshots button on toolbar
  7. click Firefox Screenshots button on toolbar again to escape the Screenshots.
  8. click the bear (background image), or any other empty space on the page

Actual result:
  Notification is shown with the following message
    Whoa! Firefox Screenshots went haywire.
    We’re not sure what just happened. Care to try again or take a shot of a different page?
  And also the following error message is shown in browser console (but this is not the scope of this issue)
    ReferenceError: ui is not defined

Expected result:
  Error should be displayed in more useful way with useful description
  1. The notification message doesn't make sense.
    a. I'm not good at English tho, is "go haywire" appropriate term here?
    b. the message doesn't say "Error" or "Bug" or something that clearly explains what happens
    c. why is it suggesting "try again" or "take a shot of a different page" ?
      it should suggest filing a bug instead, since trying again won't help
  2. The notification disappears shortly, and it's hard to figure out what it mean before it disappears.
    Notification isn't appropriate feature to use for error reporting.
    at least the error message should be displayed inside Firefox window.

**Comment by ianb** [comment]
I believe this is fixed in 9.0.0 by #2838

**Comment by arai-a** [comment]
no, this issue is about the error message and how it's displayed, not about the specific error fixed by https://github.com/mozilla-services/screenshots/commit/5d04ef1d7ac52aa181760fd5279a89e69f9691c5

**Comment by chenba** [comment]
Does this still `needs:UX` or at least copy for the new message? @johngruen?
